src/quo2/components/settings/settings_item/style.cljs

```diff
@@ -13,9 +13,10 @@
   {:flex-direction :row
    :align-items    (if (or tag description) :flex-start :center)})
 
-(def sub-container
+(defn sub-container
+  [align-action]
```

Comment:
it's best to name this as `align-items`.

Reply:
We are not aligning items here. This is to align the action button.